mainwindow.py

```diff
@@ -612,7 +612,9 @@ def populate_footprint_list(self, *_):
             )
             if detail:
                 part[4] = detail[0][2]
-                part[5] = detail[0][1]
+                if part[5] != str(detail[0][1]):
+                    part[5] = str(detail[0][1])
+                    self.store.set_stock(part[0], detail[0][1])
```

Comment:
when would this condition occur? When the database is updated while the plugin is open, without closing and reopening the plugin?

Reply:
It occur persistently after updating a database while having some projects before. Reopening the plugin doesn't help. `self.store` is project-bound and does not explicitly sync its stock quantity value with the database at all. There's just one path where it does - and that's when you use a part selector - and never afterwards. This fix is meant to address it.